spring-cloud-stream-app-generator/pom.xml

```diff
@@ -17,11 +17,11 @@
 			<plugin>
 				<groupId>org.springframework.cloud.stream.app.plugin</groupId>
 				<artifactId>spring-cloud-stream-app-maven-plugin</artifactId>
-				<version>${scs-app-maven-plugin.version}</version>
+				<version>1.0.0.BUILD-SNAPSHOT</version>
```

Comment:
This version property as well. May be you have added them to test this or is there a reason behind this?